scribble/db/seeds.rb

```diff
@@ -0,0 +1,15 @@
+# This fiTRUNCATE TABLE comments CASCADE;
+TRUNCATE TABLE comments CASCADE;
+
+ALTER SEQUENCE posts_id_seq RESTART WITH 1;
+ALTER SEQUENCE comments_id_seq RESTART WITH 1;
+
+
+INSERT INTO  (email) VALUES ('mrchensorganiccontrolroom@gmail.com' );
+
```

Comment:
Your `seeds.rb` file should only contain Ruby code. So instead of SQL, you're going to use ActiveRecord methods to populate/modify your database. Take a look at the examples that are included in the default commented out code in this file.